week08_pomdp/practice_tensorflow.ipynb

```diff
@@ -17,17 +17,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import sys, os\n",
-    "if 'google.colab' in sys.modules:\n",
-    "    %tensorflow_version 1.x\n",
-    "    \n",
+    "import os, sys\n",
+    "\n",
+    "if 'google.colab' in sys.modules:    \n",
     "    if not os.path.exists('.setup_complete'):\n",
```

Comment:
`if 'google.colab' in sys.modules and not os.path.exists('.setup_complete'):`

Reply:
ok